frontend/src/components/Header.vue

```diff
@@ -2,62 +2,62 @@
   <header class="flex items-center py-4 px-6 xl:container xl:mx-auto">
     <div class="flex items-center flex-1">
       <button class="text-gray-500 focus:outline-none lg:hidden" @click="isOpen = true">
-        <GoIcons name="Menu" class="h-6 w-6" />
+        <MenuIcon class="h-6 w-6"/>
       </button>
 
-      <SearchBar class="mx-4 lg:mx-0 sm:w-96 active:w-full" />
+      <SearchBar class="mx-4 lg:mx-0 sm:w-96 active:w-full"/>
     </div>
 
     <div class="flex items-center gap-4">
-      <div class="hover:text-gray-500 cursor-pointer" @click="setPrivateMode(!privateMode)">
-        <GoIcons v-if="privateMode !== true" name="Eye" type="outline" class="h-7 w-7" />
-        <GoIcons v-if="privateMode === true" name="EyeOff" type="outline" class="h-7 w-7" />
+      <div class="hover:text-gray-500 cursor-pointer" @click="userStore.setPrivateMode(!privateMode)">
+        <EyeIcon v-if="privateMode !== true" class="h-7 w-7"/>
+        <EyeOffIcon v-if="privateMode === true" class="h-7 w-7"/>
       </div>
 
-      <GoIcons name="Bell" type="outline" class="h-7 w-7 hover:text-gray-500 cursor-pointer" />
+      <BellIcon class="h-7 w-7 hover:text-gray-500 cursor-pointer"/>
 
       <div class="relative">
         <button
-          class="relative z-10 block h-8 w-8 rounded-full overflow-hidden shadow focus:outline-none"
-          @click="dropdownOpen = !dropdownOpen"
+            class="relative z-10 block h-8 w-8 rounded-full overflow-hidden shadow focus:outline-none"
+            @click="dropdownOpen = !dropdownOpen"
         >
           <img
-            alt="Your avatar"
-            class="h-full w-full object-cover"
-            src="https://images.unsplash.com/photo-1528892952291-009c663ce843?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=296&q=80"
+              alt="Your avatar"
+              class="h-full w-full object-cover"
+              src="https://images.unsplash.com/photo-1528892952291-009c663ce843?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=296&q=80"
```

Comment:
I guess it's a placeholder for the profile picture ? Do you have a generic one with the icons library ? Or maybe use Gravatar?

Reply:
Actually I dont think we will have avatar pictures in the end, so maybe we should just show the user name in the header